### PR TITLE
Keymap / Macro name

### PR DESCRIPTION
**Pull request by NejcZdovc**
PR based on #132

### CODE REVIEW

**Comment by NejcZdovc** [comment]
Fixed

**Comment by NejcZdovc** [comment]
Implemented new flow without errors, just placing old value back into an input.

File: src/components/keymap/header/keymap-header.component.html [hunk 1]
```diff
@@ -1,13 +1,17 @@
 <div class="row">
     <h1 class="col-xs-12 pane-title">
         <i class="fa fa-keyboard-o"></i>
-        <input class="keymap__name pane-title__name"
+        <input #name class="keymap__name pane-title__name"
+               type="text"
                value="{{ keymap.name }}"
                (change)="editKeymapName($event.target.value)"
+               (keyup.enter)="name.blur()"
```
Comment (fjozsef) [inline src/components/keymap/header/keymap-header.component.html]: Shouldn't we use renderer with `invokeElementMethod` here? I don't know. I don't have information about is it accepted like this in the template or not.

File: src/components/macro/header/macro-header.component.ts [hunk 2]
```diff
@@ -35,6 +38,12 @@ export class MacroHeaderComponent {
     }
 
     editMacroName(name: string) {
+        if (name.length === 0) {
+            this.renderer.setElementProperty(this.macroName.nativeElement, 'value', this.macro.name);
+
```
Comment (fjozsef) [inline src/components/macro/header/macro-header.component.ts]: Remove empty line.

File: src/components/keymap/header/keymap-header.component.html (hunk 1 above)
Comment (NejcZdovc) [inline src/components/keymap/header/keymap-header.component.html]: AFAIK this is ok

File: src/components/macro/header/macro-header.component.ts (hunk 2 above)
Comment (NejcZdovc) [inline src/components/macro/header/macro-header.component.ts]: Done

**Pull request merged**